server/runtime/src/test/java/org/infinispan/server/test/CountdownLatchLoggingConsumer.java

```diff
@@ -29,7 +29,28 @@ public void accept(OutputFrame outputFrame) {
       }
    }
 
+   /**
+    * Wait for the timeout interval using a sleep divisor 10. If the timeout is 10 seconds. It will check if
+    * {@linkplain #latch getCount()} is equals to 0 every second.
```

Comment:
`latch.await()` returns immediately after `latch.countDown()` is called the `count`th time, there's no need to split the wait interval in 10.

The only thing that needs to change from master is to throw an exception if `latch.await()` returns `false`.

Ok, maybe also change the method name, because when I see something like `CountdownLatchX.await()` I don't expect it to throw an exception on timeout, I expect it to return `false`. In `CheckPoint` I used `awaitStrict()`, but I'm not terribly happy with that name.